Makefile

```diff
@@ -1,56 +1,24 @@
-BIN := kube-mgmt
-PKG := github.com/open-policy-agent/kube-mgmt
-REGISTRY ?= openpolicyagent
-VERSION := 0.12.1
-ARCH := amd64
-COMMIT := $(shell ./build/get-build-commit.sh)
```

Comment:
Is the script not working as expected ?

Reply:
Didnt understand the question

Reply:
Why are we deleting the file `build/get-build-commit.sh` and instead adding the same command in here. Is that script not working as expected ?

Reply:
Because, those script are not used anywherre outside release pupeline. There is no need to pollute the project with unnecessary files. Instead al release logic is moved to makefile and pipeline.